v0/destinations/kafka/transform.js

```diff
@@ -1,3 +1,4 @@
+const cloneDeep = require("lodash/cloneDeep");
 const { getIntegrationsObj } = require("../../util");
 
 function batch(destEvents) {
```

Comment:
batch logic needs to be handled separately - internally it's not calling process(event)

Reply:
No we are handling this as discussed